Code-Standards-and-Ways-of-Working.md

```diff
@@ -0,0 +1,79 @@
+# Code standards
+
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md) (you are here)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
I'll correct the typos, in future it's more efficient if you suggest the change than comment for typos and minor changes.

Reply:
@mykeels done